From: Jan Dittberner <jandd@debian.org>
Date: Thu, 8 Oct 2015 09:25:42 -0700
Subject: disable-unsupported-sphinx-extension

Disable Sphinx extensions not available in Debian or requiring remote access.

Forwarded: not-needed
Last-Update: 2021-11-13
---
 docs/conf.py     | 32 --------------------------------
 docs/history.rst |  2 +-
 2 files changed, 1 insertion(+), 33 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index e3cf810..716c575 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -10,11 +10,8 @@ sys.path.insert(1, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
-    'jaraco.packaging.sphinx',
-    'rst.linker',
 ]
 
 # Add any paths that contain templates here, relative to this directory.
@@ -53,32 +50,3 @@ htmlhelp_basename = 'jsonpickledoc'
 latex_documents = [
     ('index', 'jsonpickle.tex', 'jsonpickle Documentation', 'David Aguilar', 'manual'),
 ]
-
-
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/3': None}
-
-
-link_files = {
-    '../CHANGES.rst': dict(
-        using=dict(GH='https://github.com'),
-        replace=[
-            dict(
-                pattern=r'(Issue #|\B#)(?P<issue>\d+)',
-                url='{package_url}/issues/{issue}',
-            ),
-            dict(
-                pattern=r'\B\+(?P<pull>\d+)',
-                url='{package_url}/pull/{pull}',
-            ),
-            dict(
-                pattern=r'^(?m)((?P<scm_version>v?\d+(\.\d+){1,2}))\n[-=]+\n',
-                with_scm='{text}\n{rev[timestamp]:%d %b %Y}\n',
-            ),
-            dict(
-                pattern=r'PEP[- ](?P<pep_number>\d+)',
-                url='https://www.python.org/dev/peps/pep-{pep_number:0>4}/',
-            ),
-        ],
-    )
-}
diff --git a/docs/history.rst b/docs/history.rst
index 8e21750..907000b 100644
--- a/docs/history.rst
+++ b/docs/history.rst
@@ -5,4 +5,4 @@
 History
 *******
 
-.. include:: ../CHANGES (links).rst
+.. include:: ../CHANGES.rst
